src/types/synchronization.d.ts

```diff
@@ -21,15 +21,27 @@ export interface MetadataPackage {
 export interface SynchronizationResult extends MetadataImportResponse {
     instance: {
         id: string;
-        name: string;
-        url: string;
+        name?: string;
+        url?: string;
     };
-    report: {
-        typeStats: any[];
+    report?: {
+        typeStats: MetadataImportStats[];
         messages: any[];
     };
 }
 
+export type SynchronizationReportStatus = "READY" | "RUNNING" | "FAILURE" | "DONE";
+
+export interface SynchronizationReport {
+    id?: string;
+    timestamp?: Date;
```

Comment:
Minor: Why don't we always set that field?

Reply:
Actually we're setting it now on the constructor, so we do not have to manually set it manually, same goes for the id.